akka-docs/src/main/paradox/scala/stream/stream-substream.md

```diff
@@ -0,0 +1,125 @@
+# Substreams
+
+Substreams are represented as `SubSource` or `SubFlow` instances, on which you can multiplex a single `Source` or `Flow`
+into a stream of streams.
+
+SubFlows cannot contribute to the super-flow’s materialized value since they are materialized later,
+during the runtime of the flow graph processing.
+
+Stages that creates substreams are listed on @ref:[Nesting and flattening stages](stages-overview.md#nesting-and-flattening-stages)
```

Comment:
`are listed in [Nesting and flattening stages]` also, they are listed here now, so a crosslink in the other direction would be good.

Reply:
Done.